include/SFML/Graphics/Shader.hpp

```diff
@@ -91,12 +91,24 @@ class SFML_GRAPHICS_API Shader : GlResource, NonCopyable
     ////////////////////////////////////////////////////////////
     Shader();
 
+    ////////////////////////////////////////////////////////////
+    /// \brief Move constructor
+    ///
+    ////////////////////////////////////////////////////////////
+    Shader(Shader&& other);
```

Comment:
Move constructors should be marked `noexcept` whenever possible.

Also, looking at the implementation this seems like it should be:
```cpp
Shader(Shader&& other) noexcept = default;
```